sqlglot/expressions.py

```diff
@@ -527,6 +527,8 @@ class Type(AutoName):
         FLOAT = auto()
         DOUBLE = auto()
         DECIMAL = auto()
+        ORA_NUMBER = "NUMBER"
```

Comment:
this is not necessary, we can just store it as a float

Reply:
But the whole point is to parse ``CREATE TABLE z (n1 NUMBER, n2 NUMBER(10), n3 NUMBER(10, 8))`` so ``NUMBER`` must be accepted as a (Oracle) data type. And I cannot use ``TokenType.NUMBER``, because that's already used for literal numbers.

And if I remove ``exp.DataType.Type.ORA_NUMBER: "REAL"`` then ``CREATE TABLE z (n1 NUMBER, n2 NUMBER(10), n3 NUMBER(10, 8))`` will not change ``NUMBER`` to ``REAL`` for sqlite.

Advice is appreciated.

Reply:
don't you just need to add a transaltion then from REAL -> NUMBER for oracle?

Reply:
right, you should be able parse NUMBER into an intermediate representation that will then convert back to NUMBER

Reply:
i can maybe take a pass at this if you would like and you can finish it up?

Reply:
That would be awesome, thanks.

Reply:
https://github.com/tobymao/sqlglot/commit/8fbcd8b3d842d85a8d423b6c8963b42278efb8a9